lib/system/alloc.nim

```diff
@@ -33,6 +33,17 @@ const
   MaxBigChunkSize = 1 shl MaxFli - 1 shl (MaxFli-MaxLog2Sli-1)
   HugeChunkSize = MaxBigChunkSize + 1
 
+  # alignments
+  MinAlignment = 4  # also min object size
```

Comment:
Better if it stays at 8 IMHO, very rarely does an object neither contain an int/pointer on 64 bits nor a floating point number.